tdms/include/field.h

```diff
@@ -331,7 +339,7 @@ class TDFieldExporter2D{
   /**
    * Allocate the arrays to hold the field
    */
-  void allocate(int nI, int nJ);
+  void allocate(int _nI, int _nK);
```

Comment:
Fun fact. Google style prefers a **trailing** underscore for private members. So it should be the other way around: the public API gets the names unadorned, and the private class workings get underscores.

I don't propose you change here though. All of our classes break this rule. (Perhaps worth enforcing globally at some stage. Low priority for now though.)